bootstrap.go

```diff
@@ -315,9 +318,25 @@ func upgradeToVer5(s Session) {
 }
 
 func upgradeToVer6(s Session) {
-	s.Execute("ALTER TABLE mysql.user ADD COLUMN `Super_priv` enum('N','Y') CHARACTER SET utf8 NOT NULL DEFAULT 'N' AFTER `Show_db_priv`")
+	_, err := s.Execute("ALTER TABLE mysql.user ADD COLUMN `Super_priv` enum('N','Y') CHARACTER SET utf8 NOT NULL DEFAULT 'N' AFTER `Show_db_priv`")
+	if err != nil {
+		if terror.ErrorNotEqual(err, infoschema.ErrColumnExists) {
+			log.Fatal(err)
+		}
+	}
+	// For reasons of compatibility, set the non-exists privilege column value to 'Y', as TiDB doesn't check them in older versions.
+	mustExecute(s, "UPDATE mysql.user SET Super_priv='Y'")
+}
+
+func upgradeToVer7(s Session) {
```

Comment:
upgradeToVer6 add a super column, upgradeToVer7 add a process column.
Use two versions because upgradeToVer6 is already there, we can't do add operation to it.